lib/api/model/model.dart

```diff
@@ -652,6 +652,7 @@ class ZulipStream {
   @JsonKey(name: 'stream_post_policy')
   ChannelPostPolicy? channelPostPolicy; // TODO(server-10) remove
   // final bool isAnnouncementOnly; // deprecated for `channelPostPolicy`; ignore
+  bool? isRecentlyActive; // TODO(server-10)
```

Comment:
```
channel: Add isRecentlyActive field
```

Since we're already not matching the order in the API doc (see e.g. https://github.com/zulip/zulip-flutter/pull/1894#discussion_r2409029149 ), I'd put this next to the related-looking field `streamWeeklyTraffic`, perhaps just above it without an empty line in between.

Similarly elsewhere in this commit.